pkgs/mime/tool/update_media_types.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+
+/// Define additions to the base apache/httpd mime set.
+///
+/// These are deltas to the base set that make sense in a Dart context
+/// (programming environments, Dart implemented web servers, ...).
+const Map<String, List<String>> mimeAdditions = {
+  'text/markdown': ['md', 'markdown'],
+  'text/x-dart': ['dart'],
+};
+
+void main() async {
+  // https://github.com/apache/httpd/blob/trunk/docs/conf/mime.types
+  const mimeTypesUrl =
+      'https://raw.githubusercontent.com/apache/httpd/refs/heads/trunk/docs/conf/mime.types';
+
+  print('Reading from $mimeTypesUrl ...');
+  final response = await http.get(Uri.parse(mimeTypesUrl));
```

Comment:
I'd consider not having the script *automatically* fetch from someone else's server. If we have some scheduling accident, we might start causing significant load.

Consider instead downloading the file into the repo manually, and generating from that file.
Or at least have a default mode where you generate from the downloaded file, so you have to pass a flag to download a new one.
Then you won't repeatedly download while tinkering with the script.